frontend/packages/knative-plugin/src/utils/create-eventsources-utils.ts

```diff
@@ -205,7 +205,7 @@ export const useEventSourceList = (namespace: string): NormalizedEventSources =>
         ...(checkAccessVal && {
           [eventSourceModel.kind]: {
             name: eventSourceModel.kind,
-            iconUrl: getKnativeEventSourceIcon(referenceForModel(eventSourceModel)),
+            iconUrl: getKnativeEventSourceIcon(eventSourceModel.kind),
```

Comment:
I think referenceForModel was a better approach.

Reply:
same as above, TBH doesn't matter here as it's handled above